### DISCUSSION TITLE
Some financial support as donation

### DISCUSSION THREAD

**Discussion by FAKHRUL1** [community/community#16673] [Projects and Issues]
Please hardly request to donate some financial support as corona effected family

**Comment by ghost** [comment]
![A094CAFA-3BCB-470B-933A-0C1BB5074C61](https://user-images.githubusercontent.com/105374724/168500549-28df7ed4-382b-47a1-8839-5d97eab6cb2e.jpeg)
![DCFC624A-D419-4459-8E81-F805CFB21BCC](https://user-images.githubusercontent.com/105374724/168500552-abc82392-815f-4b06-93e1-2c1ccc17b80d.jpeg)
